canonical-web-and-design/vanilla-framework
# Build status docs page and add labels to components

Awaiting for this [PR](https://github.com/canonical-web-and-design/vanilla-framework/pull/2416) to land and push put docs release so I can then pull in new label buttons onto the page, as they're currently missing 🤔 

## Done

- Added status labels to components inside sidebar navigation
- Added `menu-button` component docs page

## QA

- Pull code
- Run `./run `
- Open http://0.0.0.0:8101/
- Scroll down the sidebar navigation and notice new labels next to components
  - Code - Updated
  - Accordion - Updated
  - Menu button - In progress
  - Navigation - New
  - Pagination - New
- Built in progress page for Menu button
  - Added design spec and status of the component: http://0.0.0.0:8101/patterns/menu-button/

## Details

Fixes - https://github.com/canonical-web-and-design/vanilla-framework/issues/2414 and https://github.com/canonical-web-and-design/vanilla-framework/issues/2413

## Screenshots

<img width="1438" alt="Screenshot 2019-08-01 at 16 47 09" src="https://user-images.githubusercontent.com/17748020/62307726-08e9df80-b47c-11e9-99c4-94e1261ad3ec.png">

![Screenshot 2019-08-02 at 11 56 42](https://user-images.githubusercontent.com/17748020/62365698-a09f0a80-b51c-11e9-9c49-a62d1f06b9cf.png)
